src/js/node/tls.ts

```diff
@@ -1,7 +1,10 @@
 // Hardcoded module "node:tls"
 const { isArrayBufferView, isTypedArray } = require("node:util/types");
 const net = require("node:net");
-const { Server: NetServer, [Symbol.for("::bunternal::")]: InternalTCPSocket } = net;
+const {
+  Server: NetServer,
+  [Symbol.for("::bunternal::")]: [InternalTCPSocket, addServerName],
```

Comment:
We can use a shared export for these now. No need to use this symbol anymore.

Reply:
interessante, você tem algum exemplo of this? did not find about shared exports

Reply:
you could add `internal/net.ts` which would export `Symbol("kSharedInternal")`

Reply:
nice add $zig addServerName into internals and will do a followup with a refactor removing existing unnecessary symbols